shell/platform/android/io/flutter/embedding/android/FlutterActivityAndFragmentDelegate.java

```diff
@@ -1020,6 +1019,7 @@ void onTrimMemory(int level) {
         flutterEngine.getSystemChannel().sendMemoryPressureWarning();
       }
       flutterEngine.getRenderer().onTrimMemory(level);
+      flutterEngine.getPlatformViewsController().onTrimMemory(level);
```

Comment:
Note for future me. There is no @annotation or memory levels to use for protection like android has for @color.